usage.md

```diff
@@ -0,0 +1,106 @@
+
+- OptiTrack Trio Camera with Tripod and Ground
+
+- Fingertip Pressure Sensors TPS, Transmitter, USB Receiver, and Calibration Base
+
+- EMG Sensors, Probes and Receiver
+
+- USB Hub with at least 3 Ports (RealSense, OptiTrack, EMG, plus one extra...)
+
+- Optitrack Markers including placeholders, armbands, etc.
+
+- Glue, cable reel, tools, etc...
+
+## Setup
+
+#### RealSense
+
+Plug the RealSense to the computer (preferrably directly to the computer instead of going through the hub), start the
+Windows Camera app and check the image is correctly displayed. For the stereo or the motion module, check with *Intel
+RealSense Viewer*. Close the apps after checking.
+
+#### OptiTrack
+
+Connect the OptiTrack Trio to the USB Hub. Then, start *Motive*, set the ground plane in the *Calibration System Tools*
+panel under `Calibration` &rarr; `Set Ground Plane` on the right of the main window and test the OptiTrack setup with
+the desired markers. Existing markers are listed in the *Asset Pane* window accessible in the tool bar. New marker
+frames can be created by selecting the markers in the main window, rigth click and then `Rigid Body`
+&rarr; `Create From Selected Markers`. The marker label and ID can also be seen and changed in the *Asset Pane* window.
+Make sure to test the setup will all the desired markers and check if the OptiTrack system can distinguish and localize
+them well. **Let *Motive* run in the background.**
+
+#### TPS Calibration
+
+Once the TPS Transmitter is turned on and the USB Receiver is plugged into the computer directly, run *Chameleon TVR* as
+administrator and let it connect to the sensors. Select the profile (1 or 2) depending on the transmitter that is used (
+they should be labeled). Additionaly, connect the Calibration Base to the USB Hub. Then, in the tool bar, go
+on `Calibration` &rarr; `Calibrate FingerTPS System`. Specify in the GUI where the sensors are placed and begin the
+calibration process by clicking on `begin` and by following the instructions. To ensure that the calibration was
+successful, go to `C:` &rarr; `Program Files` &rarr; `Pressure Profile Systems` &rarr; `Chameleon TVR` &rarr; `setup`
+and check if the modified date of the `-cal` file corresponds to the current time. Close *Chameleon* afterwards.
+
+#### EMG
+
+Disconnect the EMG Probes from power supply such that they start blinking, turn on the EMG Receiver and connect it to
+the USB Hub.
+
+## Recording
+
+**NOTE**: `CTRL+C` might not be working on Windows. Use `CTRL+FN+B` instead. If you find yourself in debug mode in the
+terminal, you can exit that by typing `exit`.
+
+1. Start the *Anaconda Navigator*. Multiple PowerShell Prompts will have to be launched from there.
+
+2. In the first terminal (T1), run
+   ```
+   cd .\path\to\recordings\surgeon_recording\
+   python .\copy_tps_calibration.py
+   ```
+   to copy the calibration file to the correct directory. If this is unsuccessful, the calibration didn't work.
```

Comment:
copying of the calibration is time dependent. It should be done within 10 minutes otherwise the file will not be copied